finish/src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceTest.java

```diff
@@ -64,92 +41,49 @@ public void setup() {
     public void teardown() {
         client.close();
         response.close();
-        changeSystemProperty(SYSTEM_MAINTENANCE_TRUE, SYSTEM_MAINTENANCE_FALSE);
     }
 
-    @Test
-    public void testSuite() throws InterruptedException {
-        testFallbackForGet();
-        testRetryGettingSystemProperties();
-    }
-
-    //tag::javadoc[]
+    // tag::javadoc[]
     /**
-     * testFallbackForGet - test for checking if the fallback is being called correctly
-     * 1. Return system properties for a hostname when inventory service is available.
-     * 2. Make System service down and get the system properties from inventory service
-     *    when it is down.
-     * 3. Check if system properties for the specific host was returned when the inventory
-     *    service was down by:
-     *    Asserting if the total number of the system properties, when service is up, is
-     *    greater than the total number of the system properties when service is down.
-     * @return {void}
-     * @throws InterruptedException 
+     * testFallbackForGet - test for checking if the fallback is being called
+     * correctly 1. Return system properties for a hostname when inventory
+     * service is available. 2. Make System service down and get the system
```

Comment:
The formatting here got messed up :cry: 

I usually end up using html tags in my javadoc instead, then the formatter doesn't mess them up and it also looks right in the eclipse pop-up when you roll over the method name.

```
    /**
     * testFallbackForGet - test for checking if the fallback is being called correctly
     * <ol>
     * <li>Return system properties for a hostname when inventory service is available.
     * <li>Make System service down and get the system properties from inventory service
     *    when it is down.
     * <li>Check if system properties for the specific host was returned when the inventory
     *    service was down by:
     * <ul>
     *    <li>Asserting if the total number of the system properties, when service is up, is
     *    <li>greater than the total number of the system properties when service is down. 
     * </ul>
     * </ol>
```

Reply:
I'll merge the PR after all the reviews, and have you take a look at a rendered copy with the proper formatting. Will fix any issue before that.